staging/src/k8s.io/component-base/logs/options_test.go

```diff
@@ -37,7 +37,7 @@ func TestFlags(t *testing.T) {
 	want := `      --experimental-logging-sanitization   [Experimental] When enabled prevents logging of fields tagged as sensitive (passwords, keys, tokens).
                                             Runtime log sanitization may introduce significant computation overhead and therefore should not be enabled in production.
       --logging-format string               Sets the log format. Permitted formats: "text".
-                                            Non-default formats don't honor these flags: --add_dir_header, --alsologtostderr, --log_backtrace_at, --log_dir, --log_file, --log_file_max_size, --logtostderr, --one_output, --skip_headers, --skip_log_headers, --stderrthreshold, --vmodule, --log-flush-frequency.
```

Comment:
I think that `json` logging format (using zapr), doesn't support `--log-flush-frequency`. Function `FlushLogs` will call `klog.Flush`, but log doesn't support passing this call to `logr.Logger` implementation.

Reply:
Ok, I think you are right. Stdout/Stderr in Golang is also not buffered, so this was mostly for cases where klog was configured to write to files.